manifests/12_prometheus_rule.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: marketplace-alert-rules
+  namespace: openshift-marketplace
+  labels:
+    prometheus: alert-rules
+    role: alert-rules
```

Comment:
Any significance to these labels? Mainly curious as I didn't see anything consistent with our labeling in the root list of manifests.

/lgtm

Reply:
It looks like we sometimes use the `name: marketplace-operator` in some manifests. We may need to take some time at some point in the next release and populate the k8s recommended labels: https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/.

Reply:
@timflannagan that is a good point. I'm not sure why/if these labels are necessary tbh, for this PR I mainly stuck to what we were doing previously , eg in 4.4 https://github.com/operator-framework/operator-marketplace/blob/release-4.4/manifests/12_prometheus_rule.yaml#L6-L8
Mainly to make sure I'm not adding any regression here. 

That being said, I don't think putting it on our to-do list to explore the significance of these labels is a bad idea.